hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/BlockStreamConfig.java

```diff
@@ -29,18 +29,37 @@
  * Configuration for the block stream.
  * @param streamMode Value of RECORDS disables the block stream; BOTH enables it
  * @param writerMode if we are writing to a file or gRPC stream
+ * @param shutdownNodeOnNoBlockNodes whether to shutdown the consensus node if there are no block node connections
  * @param blockFileDir directory to store block files
+ * @param blockNodeConnectionFileDir directory to get the block node configuration file
  * @param compressFilesOnCreation whether to compress files on creation
+ * @param serializationBatchSize the number of items to serialize in a batch
+ * @param hashCombineBatchSize the number of items to hash in a batch
+ * @param roundsPerBlock the number of rounds per block
+ * @param deleteFilesOnDisk whether to delete files on disk
  * @param grpcAddress the address of the gRPC server
  * @param grpcPort the port of the gRPC server
  */
 @ConfigData("blockStream")
 public record BlockStreamConfig(
         @ConfigProperty(defaultValue = "BOTH") @NetworkProperty StreamMode streamMode,
         @ConfigProperty(defaultValue = "FILE") @NodeProperty BlockStreamWriterMode writerMode,
+        @ConfigProperty(defaultValue = "false") @NodeProperty boolean shutdownNodeOnNoBlockNodes,
         @ConfigProperty(defaultValue = "/opt/hgcapp/blockStreams") @NodeProperty String blockFileDir,
+        @ConfigProperty(defaultValue = "/opt/hgcapp/data/config") @NodeProperty String blockNodeConnectionFileDir,
         @ConfigProperty(defaultValue = "true") @NetworkProperty boolean compressFilesOnCreation,
+        @ConfigProperty(defaultValue = "32") @NetworkProperty int serializationBatchSize,
```

Comment:
I don't see this `serializationBatchSize` or the `deleteFilesOnDisk` properties being used anywhere, so I would suggest to remove them until we need them.